README.md

```diff
@@ -29,9 +29,20 @@ yarn add flowchart-vue
         <button type="button" @click="$refs.chart.save()">
             Save
         </button>
-        <flowchart :nodes="nodes" :connections="connections" @editnode="handleEditNode"
-                    @dblclick="handleDblClick" @editconnection="handleEditConnection" 
-                    @save="handleChartSave" ref="chart">
+        <button type="button" v-if="showRemovingConfirmation" @click="confirmRemoving">
+            Confirm removing
+        </button>
+        <button type="button" v-if="showRemovingConfirmation" @click="showRemovingConfirmation = false">
+            Reject removing
+        </button>
+        <flowchart :nodes="nodes" 
+                   :connections="connections" 
+                   :removeRequiresConfirmation
```

Comment:
Use kebab-case to align with HTML attributes: https://vuejs.org/guide/components/props.html#prop-passing-details

Reply:
Value is missing

Reply:
Ok

Reply:
Solved.